Kinvey/Kinvey/Kinvey.swift

```diff
@@ -11,9 +11,13 @@ import XCGLogger
 
 let ObjectIdTmpPrefix = "tmp_"
 
-/// Shared client instance for simplicity. Use this instance if *you don't need* to handle with multiple Kinvey environments.
+/**
+ Shared client instance for simplicity. Use this instance if *you don't need* to
```

Comment:
Better to say something along these lines - 

"The `sharedClient` is set to the default client instance that you initialize with `Kinvey.init`. All methods that use a `client` will default to this instance. If you intend to use multiple backend apps or environments, you should override this default by providing a separate `Client` instance."